vms/vms/settings.py

```diff
@@ -63,8 +63,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'vmsadmin',
-        'PASSWORD': '0xdeadbeef',
+        'USER': '',
+        'PASSWORD': '',
```

Comment:
@tapasweni-pathak Are you referring to the authentication issue on local computer? I don't know why the credentials were used. Because one can set their own username, password and run it. There are no errors on local if done that way. Are there errors in virtualbox setup if this is removed?